--- ./lib/gdi/picload.cpp	2016-05-30 12:51:52.680823147 +0200
+++ ./lib/gdi/picload.cpp	2016-05-30 12:53:03.453630382 +0200
@@ -616,7 +616,7 @@
 	int cmaps;
 	int extcode;
 
-	gft = DGifOpenFileName(filepara->file);
+	gft = DGifOpenFileName(filepara->file, NULL);
 	if (gft == NULL)
 		return;
 	do
@@ -687,11 +687,11 @@
 	}
 	while (rt != TERMINATE_RECORD_TYPE);
 
-	DGifCloseFile(gft);
+	DGifCloseFile(gft, NULL);
 	return;
 ERROR_R:
 	eDebug("[ePicLoad] <Error gif>");
-	DGifCloseFile(gft);
+	DGifCloseFile(gft, NULL);
 }
 
 //---------------------------------------------------------------------------------------------
