--- ./lib/gdi/picload.cpp	2016-05-27 10:29:57.438135760 +0200
+++ ./lib/gdi/picload.cpp	2016-05-27 10:30:23.474423021 +0200
@@ -612,7 +612,7 @@
 	int cmaps;
 	int extcode;
 
-	gft = DGifOpenFileName(filepara->file);
+	gft = DGifOpenFileName(filepara->file, NULL);
 	if (gft == NULL)
 		return;
 	do
@@ -683,11 +683,11 @@
 	}
 	while (rt != TERMINATE_RECORD_TYPE);
 
-	DGifCloseFile(gft);
+	DGifCloseFile(gft, NULL);
 	return;
 ERROR_R:
 	eDebug("[Picload] <Error gif>");
-	DGifCloseFile(gft);
+	DGifCloseFile(gft, NULL);
 }
 
 //---------------------------------------------------------------------------------------------
